core/src/main/scala/org/apache/spark/shuffle/sort/SortShuffleManager.scala

```diff
@@ -124,6 +119,9 @@ private[spark] class SortShuffleManager(conf: SparkConf) extends ShuffleManager
       endPartition: Int,
       context: TaskContext,
       metrics: ShuffleReadMetricsReporter): ShuffleReader[K, C] = {
+    val shuffleIoPlugin = conf.get(SHUFFLE_IO_PLUGIN_CLASS)
```

Comment:
was there a reason why this was changed from a private variable?

Reply:
This wasn't possible because the ExternalShuffleService leveraged the BlockManager which was not initialized